tasks/clean/shutdown_vms.yml

```diff
@@ -10,7 +10,7 @@
       include_tasks: tasks/clean/shutdown_vm.yml
       vars:
           vms: "{{ item }}"
-      with_items: "{{ ovirt_vms }}"
```

Comment:
Please remove all ocurances of ovirt modules calls. not only `ovirt_vm`, here is also `ovirt_vms_facts`, which should be renamed to `ovirt_vm_facts`.

Reply:
removed